### ISSUE TITLE
Example with Save

### ISSUE THREAD

**Issue by gmareater** [mikemichaelis/dnc#2]
hi
first: thanx for your great example
second: could you improve your example with, "Save"

currently in our situation (base on your example with angular 6.0), save does not work correctly and throws exception

thanx

**Comment by WisdomCorp** [comment]
Yes. I am also facing same issue. Query works fine, when we try to save it, it throws error. 

If you can put a save example it would be nice.
